CHANGES.txt

```diff
@@ -1193,7 +1193,7 @@ Features
   configuring the section pages (Issue #1980)
 * For themers: Each ``post`` are now asssociated with section_color,
   section_link, and section_name (Issue #1980)
-* Each new section page has a auto-assigned color based on shifting
+* Each new section page has an auto-assigned color based on shifting
```

Comment:
Let’s not bother modifying an ancient changelog line.

Reply:
ok